### DISCUSSION TITLE
How to remove theme toggle button from tab line?

### DISCUSSION THREAD

**Discussion by mgutz** [NvChad/NvChad#1604] [Q&A] [answered]
I use background transparency so I don't need a dark/light theme toggler. How do I configure NvChad  to remove the toggle theme button?

**Comment by siduck** [comment, marked as answer]
read this section https://nvchad.com/config/nvchad_ui 


So all you have to do is make the button function return only the closeall btn variable. ( u will understand this once u read the docs )

**Reply by mgutz** [reply]
I read that section before asking. Your hint prompted me to be more thorough.

**Comment by zr00da** [comment]
I understand the reasoning behind the many "figure it out" suggestions in this repo, but finding the exact solution here would still have taught me that methods [such as this](https://github.com/NvChad/ui/blob/v3.0/lua/nvchad/tabufline/modules.lua#L107) can be overwritten by name inside the `M.ui.<module>.modules` config structure and saved me an hour. Maybe I just missed it in the fine documentation, day 1 of converting to vim is no joke 🤯

For the reader who's passing out mere steps before the validating embrace of editing peaks, here's a friendly push:

```lua
-- chadrc.lua
M.ui = {
  tabufline = {
    modules = {
      btns = function()
        local closeAllBufs = btn(" 󰅖 ", "CloseAllBufsBtn", "CloseAllBufs")
        return closeAllBufs
      end,
    }
  }
}
